src/utils/AsyncStorage.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+export const storeData = async (key, value) => {
```

Comment:
buggy code 

`WARN  [AsyncStorage] The value for key ""radius"" is not a string. This can lead to unexpected behavior/errors. Consider stringifying it.`